pkg/model/initializers.go

```diff
@@ -36,6 +37,7 @@ const (
 	RwkvBackend            = "rwkv"
 	WhisperBackend         = "whisper"
 	StableDiffusionBackend = "stablediffusion"
+	LangChainHuggingFace   = "langchain-huggingface"
```

Comment:
Documentation needs to be updated too: https://localai.io/model-compatibility/index.html#model-compatibility-table

Reply:
Done